code/src/components/labyrinth.css

```diff
@@ -0,0 +1,124 @@
+.background {
+  display: flexbox;
+  flex-direction: column;
+  gap: 10px;
```

Comment:
We didn't know we could use only "gap" in css. Will definitely use this in the future!!